dissect/target/plugins/apps/vpn/openvpn.py

```diff
@@ -15,6 +18,7 @@
     ("string", "ca"),
     ("string", "cert"),
     ("string", "key"),
+    ("boolean", "redacted"),
```

Comment:
```suggestion
    ("boolean", "key_redacted"),
```
Maybe makes it a little bit clearer?